core/tests/snapshots.rs

```diff
@@ -471,17 +468,6 @@ fn test_concurrent_snapshot_packaging(
         }
     }
 
-    // Purge all the outdated snapshots, including the ones needed to generate the package
```

Comment:
The purge function no longer purges the PRE-stage bank snapshot dirs.  So this test is no longer valid.

Reply:
Not in this PR any more.